bin/install

```diff
@@ -47,11 +47,21 @@ install_default_npm_packages() {
   }
 
   local args=()
-
   local line="" aux=()
+  local printed_empty_lines_warn=0 
   while read -r line; do
-    pkg_lines+=("$line")
+    if [[ -z "$line" ]]; then
+      # Ignore empty lines and print warning
+      if [ "$printed_empty_lines_warn" -eq 0 ]; then
+        echo "$(colored $YELLOW WARNING:) There are empty line ${pkgs_file}"
+        printed_empty_lines_warn=1
+      fi
```

Comment:
The warning is unnecessary if the empty line is skipped. It's warning about something that is not actually a problem anymore because of your change. I'd say just `continue` and get rid of this warning

Reply:
I removed the warning message. 👌